x-pack/plugins/security_solution/public/timelines/components/timeline/helpers.tsx

```diff
@@ -253,3 +261,94 @@ export const focusUtilityBarAction = (containerElement: HTMLElement | null) => {
 export const resetKeyboardFocus = () => {
   document.querySelector<HTMLAnchorElement>('header.headerGlobalNav a.euiHeaderLogo')?.focus();
 };
+
+interface OperatorHandler {
+  field: string;
+  isExcluded: string;
+  value: string | number | Array<string | number>;
+}
+
+export const handleISOperator = ({
```

Comment:
consider using the following casing for this function name: `handleIsOperator`

Reply:
changed